misc/sample_application/image_builder/setup_hmi_vm.sh

```diff
@@ -11,8 +11,9 @@ function umount_directory() {
 }
 
 function update_package_info() {
-    apt update
-    apt install python3 python3-pip net-tools python3-matplotlib
+    apt update && apt install python3 python3-pip \
+	    net-tools python3-matplotlib -y
+    source ~/.bashrc
```

Comment:
For `.bashrc`, please include that file at the beginning of this script so that other helper functions can have the proxies set up as well.

Reply:
@junjiemao1 move the source .bashrc to the top of the script so that it can be used by the full script.